x/tls_termination_test.go

```diff
@@ -35,51 +35,83 @@ func TestDoesRequestSatisfyTermination(t *testing.T) {
 		assert.EqualValues(t, http.StatusBadGateway, res.Code)
```

Comment:
The diff here is very difficult to read for me because it looks like the test order has been changed? Could you please do one of the two:

- Reverse the order change and append your new test cases to the bottom and change the test cases you might have changed
- Comment on every existing test change why it was changed

Reply:
I have ordered tests like original and have placed 2 new tests at the bottom. for changed tests I have made a comment why I have changed them.

Reply:
I have previously reordered tests to follow the actual flow of steps execution, but it made PR reviewing hard. sorry for that.

Reply:
All good, thank you for revisiting!